src/app.js

```diff
@@ -0,0 +1,52 @@
+//- Kijken of alles goed gekoppeld is met console.log
+// console.log("Welkom");
+
+//- Hiermee koppelen we ons app.js bestand aan package.json en axios
+import axios from "axios" ;
+
+function handleField(e) {
+    e.preventDefault();
+    fetchCountryInformation(inputField.value);
+}
+
+const inputField = document.getElementById("search-field");
+inputField.addEventListener("keyup", (e) => e.keyCode === 13);
```

Comment:
`keyCode` is deprecated. Het is beter om gebruik te maken van `key` of `code`.
https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/keyCode